src/gui/tracks/InstrumentTrackView.cpp

```diff
@@ -140,8 +140,14 @@ InstrumentTrackView::InstrumentTrackView( InstrumentTrack * _it, TrackContainerV
 	connect(midiRackAction, SIGNAL(triggered()),
 		this, SLOT(toggleMidiCCRack()));
 
+	QPalette::ColorRole role;
+#if (QT_VERSION >= QT_VERSION_CHECK(5,13,0))
+	role = QPalette::Window;
+#else
+	role = QPalette::Background;
+#endif
 	m_activityIndicator = new FadeButton( QApplication::palette().color( QPalette::Active,
-							QPalette::Background),
+							role),
```

Comment:
`QPalette::Window` exists in all versions of Qt 5. Just use that exclusively. (The version in which something becomes deprecated is not necessarily the same as the version in which the alternative is introduced.)